codemeta.json

```
@@ -0,0 +1,86 @@
+{
+  "@context": "https://raw.githubusercontent.com/codemeta/codemeta/master/codemeta.jsonld",
+  "@type": "Code",
+  "author": [
+    {
+      "@id": "http://orcid.org/0009-0000-0056-4322",
+      "@type": "Person",
+      "email": "pedroa3@al.insper.edu.br",
+      "name": "Pedro2712",
+      "affiliation": "INSPER"
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "Luis Filipe Carrete",
+      "affiliation": "INSPER"
+    },
+    {
+      "@id": "https://orcid.org/0009-0008-1237-3353",
+      "@type": "Person",
+      "email": "ricardorr7@al.insper.edu.br",
+      "name": "RicardoRibeiroRodrigues",
+      "affiliation": "INSPER"
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "leonardo.malta",
+      "affiliation": "INSPER"
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "Manuel Castanares",
+      "affiliation": "INSPER"
+    },
+    {
+      "@id": "https://orcid.org/0009-0001-5943-0481",
+      "@type": "Person",
+      "email": "renatolf1@al.insper.edu.br",
+      "name": "renatex333",
+      "affiliation": "INSPER"
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "Enrico",
+      "affiliation": "INSPER"
+    },
+    {
+      "@id": "",
+      "@type": "Person",
+      "email": "",
+      "name": "leonardodma",
+      "affiliation": "INSPER"
+    },
+    {
+      "@id": "https://orcid.org/0000-0001-6263-121X",
+      "@type": "Person",
+      "email": "fabriciojb@insper.edu.br",
+      "name": "Fabricio J Barth",
+      "affiliation": "INSPER"
+    },
+    {
+      "@id": "https://orcid.org/0009-0005-1883-8703",
+      "@type": "Person",
+      "email": "jorascco@al.insper.edu.br",
+      "name": "Joras Oliveira",
```

trocar o a de Joras para á, com "Jorás"
```suggestion
      "name": "Jorás Oliveira",
```